enterprise/api_client/ecommerce.py

```diff
@@ -65,3 +70,31 @@ def get_course_final_price(self, mode, currency='$', enterprise_catalog_uuid=Non
         if price != mode['min_price']:
             return format_price(price, currency)
         return mode['original_price']
+
+    def create_ecommerce_order_for_manual_course_enrollment(  # pylint: disable=invalid-name
```

Comment:
nit: ecommerce can be removed from this function name; it's redundant since this is the ecommerce api client, and it will make the function name slightly more concise